protostar/commands/test/test_case_runners/fuzz_test_case_runner.py

```diff
@@ -8,27 +8,27 @@
     FuzzInputExceptionMetadata,
 )
 from protostar.commands.test.test_case_runners.test_case_runner import TestCaseRunner
+from protostar.commands.test.test_environment_exceptions import ReportedException
 from protostar.commands.test.test_results import (
     FailedFuzzTestCaseResult,
     FailedTestCaseResult,
     FuzzResult,
     PassedFuzzTestCaseResult,
     TestResult,
 )
-from protostar.commands.test.test_environment_exceptions import ReportedException
 
 
 class FuzzTestCaseRunner(TestCaseRunner[FuzzTestExecutionResult]):
```

Comment:
Note: the subclasses of `TestCaseRunner` are going to be removed here: https://github.com/software-mansion/protostar/commit/8b284ef3ba4009938880e7f58b0e99f645a75580